### PR TITLE
Fix #6563 Expression documentation and EMX doc link

### LINKED ISSUE #6563: EMX Documentation refers to non-existing Javascript Expressions section
### How to Reproduce
- Go to https://molgenis.gitbooks.io/molgenis/content/user_documentation/ref-emx.html
- Look up the validationExpressions description

### Expected behavior
- There is a working (reference for) the documentation how to make Javascript Expressions

### Observed behavior
- There is a [[Javascript Expressions]] marker

### PR DESCRIPTION
**Pull request by dennishendriksen**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: docs/user_documentation/ref-emx.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ true/false to indicate a readOnly attribute
 ability to tag the data referring to the tags sections, described below
 
 ### validationExpression 
-javascript validation expression that must return a bool. Must return true if valid and false if invalid. See for a syntax description the section [[Javascript Expressions]]
+javascript validation expression that must return a bool. Must return true if valid and false if invalid. See for a syntax description the section [Expressions](ref-expressions)
```
Comment (Mark-de-Haan) [inline docs/user_documentation/ref-emx.md]: See for a syntax description the section [Expressions](ref-expressions) -> See the [Expressions](ref-expressions) section for a syntax description.

File: docs/user_documentation/ref-expressions.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+attribute values e.g. to determine whether values are valid. The expression format is based on the 
+[Magma JavaScript API](http://wiki.obiba.org/display/OPALDOC/Magma+Javascript+API).
+
+### Examples
+The following expression returns true when the value of attribute 'myAttributeName' of an entity only contains
+alphanumberic characters: 
+```js
+$('myStringAttributeName').matches(/^[a-z0-9]+$/i).value()
+```
+Is this example:
```
Comment (Mark-de-Haan) [inline docs/user_documentation/ref-expressions.md]: Is this example: -> In this example:

File: docs/user_documentation/ref-emx.md (hunk 1 above)
Comment (dennishendriksen) [reply docs/user_documentation/ref-emx.md]: Agreed. Feel free to make these minor edits yourself.

File: docs/user_documentation/ref-expressions.md (hunk 2 above)
Comment (dennishendriksen) [reply docs/user_documentation/ref-expressions.md]: Agreed. Feel free to make these minor edits yourself.

**Pull request merged**